lib/system_registry/task.ex

```diff
@@ -0,0 +1,52 @@
+defmodule SystemRegistry.Task do
+  @moduledoc """
+
+  Creates a process that executes a function anytime the value of a given
+  system_registry scope changes.
+
+  For example, lets say we want to perform a quick task any time the ipv4 address
+  value of the wireless interface changes.
+
+    worker(SystemRegistry.Task, [
+      [:state, :network_interface,  "wlan0", :ipv4_address],
+      fn(_old, _new) ->
+        # Do something
+      end])
+
```

Comment:
You had a comment in the commit message about using this sparingly that I think should be here as well. Most importantly, this helper is only good for invoking functions that have side effects since there's no state that's passed through.

Reply:
I was just considering if we should make it that the function thats passed in has `({new, old}, state)` and that the return of the function persists the state. I was considering this because I kinda wanted a way to use this only on the first time it changes.

Reply:
Relaying commentary from slack: Seems like once state is added, you're going to handle non-SystemRegistry things as well and consequently not use this.

Also, I'm skeptical on whether a first-time only usecase really exists. Willing to be educated, though, and since it sounded on Slack that state wouldn't be added for now, this is probably moot.